### PR TITLE
Mobilpay.ro Payment Gateway recipe

### PR DESCRIPTION
**Pull request by birkof**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: birkof/netopia-mobilpay-bundle/1.0/config/netopia_mobilpay/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Ignore everything
```
Comment (Nyholm) [inline birkof/netopia-mobilpay-bundle/1.0/config/netopia_mobilpay/.gitignore]: Can you tell me why this file is needed?

File: birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+netopia_mobilpay:
+    payment_url:        '%env(NETOPIA_MOBILPAY_PAYMENT_URL)%'
+    public_cert:        '%env(NETOPIA_MOBILPAY_PUBLIC_CERT)%'
+    private_key:        '%env(NETOPIA_MOBILPAY_PRIVATE_KEY)%'
+    signature:          '%env(NETOPIA_MOBILPAY_SIGNATURE)%'
+    return_url:         'netopia_mobilpay_return_url'
```
Comment (Nyholm) [inline birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml]: Shouldnt these routes be registered?

File: birkof/netopia-mobilpay-bundle/1.0/config/netopia_mobilpay/.gitignore (hunk 1 above)
Comment (Nyholm) [reply birkof/netopia-mobilpay-bundle/1.0/config/netopia_mobilpay/.gitignore]: Could you write a comment about that? These generic comments does not really help.

File: birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml (hunk 2 above)
Comment (Nyholm) [reply birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml]: Okey, Maybe create a routing file with those defined? (But just pointing to `/`)?

File: birkof/netopia-mobilpay-bundle/1.0/config/netopia_mobilpay/.gitignore (hunk 1 above)
Comment (birkof) [reply birkof/netopia-mobilpay-bundle/1.0/config/netopia_mobilpay/.gitignore]: Documentation is in progress for the entire bundle, which will be available in few hrs, I hope...

File: birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml (hunk 2 above)
Comment (birkof) [reply birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml]: Nice hint! Lemme switch to...

Comment (birkof) [reply birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml]: @Nyholm changes pushed 👍

File: birkof/netopia-mobilpay-bundle/1.0/manifest.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "bundles": {
+        "birkof\\NetopiaMobilPay\\NetopiaMobilPayBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "gitignore": [
+        "/%CONFIG_DIR%/netopia_mobilpay/*.key",
+        "/%CONFIG_DIR%/netopia_mobilpay/*.cer"
```
Comment (Nyholm) [inline birkof/netopia-mobilpay-bundle/1.0/manifest.json]: Good!

You can remove your `.gitignore` now.

Comment (birkof) [reply birkof/netopia-mobilpay-bundle/1.0/manifest.json]: Done!

**Comment by Nyholm** [comment]
No, do the other way around. 
Remove your .gitignore instead. It is way more convenient to have all "ignore rule" at the root.

**Comment by birkof** [comment]
I think if I'll remove .gitignore file also the folder would not longer be available. Maybe with a .gitkeep file in?

**Comment by Nyholm** [comment]
True, just add a an empty .gitignore file will make sure the folder is created

**Comment by birkof** [comment]
All done!

**Pull request merged**